setup.py

```diff
@@ -47,6 +47,8 @@
           'xarray>=0.12.3',
           'pandas>0.20.0',
           'deprecated>=1.2.12',
+          'oschmod; sys_platform == "win32"',
+          'pywin32; sys_platform == "win32"',
```

Comment:
I don't think it's necessary to list pywin32 explicitly as dependency, we just use oschmod and pywin32 is one of it's implementation details, right?

Reply:
Yes, we should be able to leave out pywin32.
When pip resolves dependencies it should know whether the platform needs it or not.

Will change it and test it.

Reply:
Left out pywin32

oschmod is a required dependency for Windows, but not for the other platforms.

I added oschmod as a new item in extra_require, which also called "oschmod"
This way it can be optionally installed on other platforms more easily, with a single pip-install command:
```
pip-install .[dev,oschmod]
```